torch/testing/_internal/opinfo_helper.py

```diff
@@ -4,21 +4,21 @@
 
 import torch
 from torch.testing._internal.common_cuda import (TEST_CUDA)
-from torch.testing._core import _dispatch_dtypes
-from torch.testing import (all_types_and_complex_and,
-                           all_types_and_complex,
-                           all_types_and_half,
-                           all_types,
-                           complex_types,
-                           floating_and_complex_types,
-                           floating_types_and_half,
-                           floating_types,
-                           integral_types,
-                           floating_types_and,
-                           floating_and_complex_types_and,
-                           integral_types_and,
-                           all_types_and,
-                           )
+from torch.testing._internal.dtype_getters import (all_types_and_complex_and,
```

Comment:
This can be more easily formatted:

```
import \
  ( ... )
```

without having to space all the imported elements so far to the right of the page

Reply:
I try to refrain form pure formatting changes as much as possible, because they only create noise.